improvement-exercise/exercise-one.js

```diff
@@ -1,6 +1,5 @@
 function calculateAverage(numbers) {
   const sum = numbers.reduce((add, number) => add + number, 0);
-  const average = sum / numbers.length;
-  return average;
+  return average = sum / numbers.length;
```

Comment:
this can be much better since you don't need the average variable
```suggestion
  return  sum / numbers.length;
```